### PR TITLE
Get Emoticon hotkeys

### PR DESCRIPTION
**Pull request by eddiemoore**

### CODE REVIEW

File: test/emoticons.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+/* global describe, it, beforeEach, afterEach */
+import chai, { expect } from 'chai'
```
Comment (cbas) [inline test/emoticons.js]: FYI this chai/expect/sinon boilerplate can be avoided by using a mocha.opts file. Example: https://github.com/cbas/handsfree/tree/master/test

File: test/emoticons.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  })
+
+  it('should get a list of emoticons', () => {
+    stub = sinon.stub(emoticons._dinghy, 'send')
+    emoticons.list()
+    const packet = {
+      type: PacketType.GET_EMOTICON_HOTKEYS,
+      fields: {}
+    }
+    expect(stub).to.have.been.called
```
Comment (cbas) [inline test/emoticons.js]: Redundant?

File: test/emoticons.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    const hotkey_list = ['(someemoticon)']
+    emoticons.getEmoticon(hotkey_list)
+    const packet = {
+      type: PacketType.GET_EMOTICON,
+      fields: {
+        hotkey_list,
+        use_json_data: true
+      }
+    }
+    expect(stub).to.have.been.called
```
Comment (cbas) [inline test/emoticons.js]: Redundant?

Comment (eddiemoore) [inline test/emoticons.js]: Probably yeah :+1:

**Pull request merged**